homeassistant/components/remote/services.yaml

```diff
@@ -2,52 +2,85 @@
 
 turn_on:
   description: Sends the Power On Command.
+  target:
   fields:
     entity_id:
       description: Name(s) of entities to turn on.
       example: "remote.family_room"
```

Comment:
Target selectors remove the need for having the entity_id fields. This is now a duplicate?

Reply:
Done.